src/main/kotlin/kotlinx/kover/engines/intellij/IntelliJCoverage.kt

```diff
@@ -124,22 +92,35 @@ private fun readCounterValuesFromXml(file: File): Map<VerificationValueType, Int
 
 
 private fun Task.checkRule(counters: Map<VerificationValueType, Int>, rule: VerificationRule): String? {
-    val minValue = rule.minValue
-    val maxValue = rule.maxValue
+    val boundsViolations = rule.bounds.mapNotNull { it.check(counters) }
+
+    val ruleName = if (rule.name != null) "'${rule.name}' " else ""
+    return if (boundsViolations.size > 1) {
+        "Rule ${ruleName}violated for '${project.name}':" + boundsViolations.joinToString("\n  ", "\n  ")
```

Comment:
Shouldn't there be space?

```suggestion
        "Rule ${ruleName} violated for '${project.name}':" + boundsViolations.joinToString("\n  ", "\n  ")
```

Reply:
Too implicit... because in this case if there is no custom name there will be two spaces between words

Reply:
It's possible to add some `if`s though